diagnostic.md

```diff
@@ -98,7 +109,7 @@ object.
   "name": "Krüller"
 }
 ```
-
+curl --include --request PATCH http://www.example.com/donuts/3
```

Comment:
That's a good start. In order to change something, we have to say _which_ thing we want to change (which you did), and also what we want to change it _to_. Checkout this solution:
```
curl --include --request "PATCH" "http://example.com/donuts/42" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "Krüller",
    }
  }'
```